homeassistant/components/motion_blinds/config_flow.py

```diff
@@ -72,6 +74,18 @@ def async_get_options_flow(config_entry) -> OptionsFlowHandler:
         """Get the options flow."""
         return OptionsFlowHandler(config_entry)
 
+    async def async_step_dhcp(self, discovery_info: dhcp.DhcpServiceInfo) -> FlowResult:
+        """Handle discovery via dhcp."""
+        mac_address = format_mac(discovery_info.macaddress).replace(":", "")
+        await self.async_set_unique_id(mac_address)
+        self._abort_if_unique_id_configured(updates={CONF_HOST: discovery_info.ip})
+
+        short_mac = format_mac(discovery_info.macaddress)[-8:]
```

Comment:
```suggestion
        short_mac = mac_address[-6:].upper()
```

Usually we just use the last 6